Assets/Scripts/UI/PlayerHUDController.cs

```diff
@@ -373,9 +379,27 @@ private void SetHitmarkScale(float scale)
         crosshairMaterial.SetFloat("_HitMarkerRadius", scale);
     }
 
+    private void SetCrosshairRadius(float radius)
+    {
+        crosshairMaterial.SetFloat("_Radius", radius);
+    }
+
+    public void CrossHairDetonationAnimation()
+    {
+        if (LeanTween.isTweening(crosshairRadiusTween))
+        {
+            LeanTween.cancel(crosshairRadiusTween);
+            crosshairMaterial.SetFloat("_Radius", initialCrosshairRadius);
+        }
+        crosshairRadiusTween = LeanTween.value(crosshair.gameObject, SetCrosshairRadius, initialCrosshairRadius, 0f, 0.2f)
```

Comment:
This animation was a little too brief, perhaps increase its duration by 100 ms or so?

Reply:
I was afraid the feedback wouldn't be fast enough, but after testing it seems to be ok, so I set it to 0.3 👍